src/jwplayer/players/DesktopArticleVideoPlayer/Attribution.module.css

```diff
@@ -8,10 +8,12 @@
 
 .usernameLink {
 	color: rgba(var(--theme-page-text-color--rgb), 0.75);
+	cursor: pointer;
 }
 
 .iconLink {
 	color: rgba(var(--theme-page-text-color--rgb), 0.75);
+	cursor: pointer;
```

Comment:
what's the point of adding `cursor: pointer` here? These are `<a>` elements with `href` which by default have pointer cursor.

Reply:
Yeah, we can remove it. I was not aware that they have this by default